CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Current (in progress)
 
+- Fix FTP links in markdown [#2448](https://github.com/opendatateam/udata/pull/2448)
```

Comment:
No need for a CHANGELOG entry for once :-) This is not a feature of udata2, just a fix to make it work like udata1.